sunpy/data/_sample.py

```diff
@@ -46,6 +46,7 @@
     "GBM_TIMESERIES": "glg_cspec_n5_110607_v00.pha",
     "RHESSI_TIMESERIES": "hsi_obssumm_20110607_025.fits",
     "NORH_TIMESERIES": "tca110607.fits",
+    "LOFAR_IMAGE": "LOFAR_70MHZ_20190409_131136.fits",
```

Comment:
Was this added (or already exist in the sample data repo?

Reply:
yep it exists - it was added here https://github.com/sunpy/sample-data/pull/14